deploy.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 
+set -x
```

Comment:
In CodeBuild, the automatic credential discovery actually requires that the `AWS_CONTAINER_CREDENTIALS_RELATIVE_URI` environment variable be set to what CodeBuild sets it[1].

[1] https://docs.aws.amazon.com/codebuild/latest/userguide/troubleshooting.html#troubleshooting-versions

Reply:
Nevermind; I was misremembering what set x/u/e does.